### PR TITLE
Add license

### LINKED ISSUE #63: Update tag and README to latest release
The README is still talking about MEI 3 as the latest release. That could be confusing ;)

EDIT: Also the one and only release tag is 3.0.0

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR adds a license as decided during MEI Board meeting.
Also it updates the readme file.

Closes #63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # The MEI Sample Encodings
 
-The MEI Sample Encodings contain a wide range of MEI-encoded music files. The intent is to demonstrate the use of MEI in real-world contexts. The current MEI Release is 3.0.0. Samples from previous releases may be found in the 'legacy' directory.
+The MEI Sample Encodings contain a wide range of MEI-encoded music files. The intent is to demonstrate the use of MEI in real-world contexts. The current MEI Release is 4.0.1. Samples from previous releases may be found in the `legacy` and `MEI_3.0` directories.
```
Comment (musicEnfanthen) [inline README.md]:
```suggestion
The MEI Sample Encodings contain a wide range of MEI-encoded music files. The intent is to demonstrate the use of MEI in real-world contexts. The current MEI Release is 4.0.1. Samples from previous releases may be found in the `MEI_3.0` and `legacy` directories.
```

File: README.md [hunk 2]
```diff
@@ -12,4 +14,10 @@ We accept "pull requests" from GitHub as an easy way to contribute your own file
 
 This will notify us of your desire to add files to this repository, and we will review your additions before accepting them.
 
-If you are not comfortable with this process but would still like to contribute, please get in touch with the MEI Technical Team. You can e-mail andrew.hankinson /at/ gmail.com.
+If you are not comfortable with this process but would still like to contribute, please get in touch with the MEI Technical Team.
```
Comment (musicEnfanthen) [inline README.md]:
```suggestion
If you are not comfortable with this process but would still like to contribute, please get in touch with the [MEI Technical Team](https://music-encoding.org/community/technical-team.html).
```

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]

**Comment by rettinghaus** [comment]
@musicEnfanthen thank you for the review.

**Review by musicEnfanthen — APPROVED** [review]

**Comment by craigsapp** [comment]
I will point out that the file https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Scarlatti_Sonata_in_C_major.mei

is a derivative electronic edition of this file https://github.com/craigsapp/scarlatti-keyboard-sonatas/blob/master/kern/L001K514.krn

which has a more restrictive license: 
https://github.com/craigsapp/scarlatti-keyboard-sonatas/blob/master/LICENSE.txt
https://creativecommons.org/licenses/by-nc-sa/4.0

**Comment by th-we** [comment]
Other files where I sense licensing problems because they are not public domain:

* [Ives_TheCage.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Ives_TheCage.mei) († 1954)
* [McFerrin_Don't_worry.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/McFerrin_Don't_worry.mei) (* 1950)

There are some arrangements that might not be kosher, either, e.g. the ones by Michel Rondeau, which seem to be CC-BY:

* [Bach-J-C_Fughette_Gmaj.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Bach-J-C_Fughette_Gmaj.mei)
* [Bach-J-C_Fughette_No2.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Bach-J-C_Fughette_No2.mei)
* [Bach_BrandenburgConcert_No.4_I.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Bach_BrandenburgConcert_No.4_I.mei)
* [Bach_BrandenburgConcert_No.4_II.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Bach_BrandenburgConcert_No.4_II.mei)
* [Beethoven_Hymn_to_joy.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Beethoven_Hymn_to_joy.mei)
* [Berlioz_Symphony_op25.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Berlioz_Symphony_op25.mei)
* [Czerny_StringQuartet_d.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Czerny_StringQuartet_d.mei)
* [Gluck_CheFaroSenzaEuridice.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Gluck_CheFaroSenzaEuridice.mei)
* [Handel_Arie.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Handel_Arie.mei)
* [Handel_Messias.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Handel_Messias.mei)
* [Hopkins_GatherRoundTheChristmasTree.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Hopkins_GatherRoundTheChristmasTree.mei)
* [Hummel_Concerto_for_trumpet.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Hummel_Concerto_for_trumpet.mei)
* [Kirnberger_FugeInEFlatMajor.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Kirnberger_FugeInEFlatMajor.mei)
* [Lully_LaDescenteDeMars.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Lully_LaDescenteDeMars.mei)
* [Praetorius_PuerNobisNascitur.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Praetorius_PuerNobisNascitur.mei)
* [Schutz_Jubilate_Deo.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Schutz_Jubilate_Deo.mei)
* [Telemann_Concert.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Telemann_Concert.mei)
* [Telemann_Suite.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Telemann_Suite.mei)

But I'm not sure if the term "arranger" was used technically correct or if he is really acting as an engraver rather than an arranger. [Saint-Saens_LeCarnevalDesAnimaux.mei](https://github.com/music-encoding/sample-encodings/blob/main/MEI_4.0/Music/Complete_examples/Saint-Saens_LeCarnevalDesAnimaux.mei) is however a true arrangement by Martin Packham (* 1978, no license specified).

**Pull request merged**